k8s/gatekeeper/Makefile

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SHELL                           := /bin/bash
+CONTAINER_CMD         	        ?= docker
+USER_ID                         = $(shell id -u)
+DOCKER_CMD_RUN_OPTS             ?= -u $(USER_ID)
+CONTAINER_CMD_RUN_OPTS          ?= $(if $(findstring docker,$(CONTAINER_CMD)),$(DOCKER_CMD_RUN_OPTS),)
+
+export DEPLOYMENT_ENV           ?= local
+# space delimited of Key:Value pairs
+COMMON_LABELS                   := instance:${DEPLOYMENT_ENV}
+export DOMAIN                   ?= ${DEPLOYMENT_ENV}.trustbloc.dev
+export BLOC_DOMAIN		        ?= orb-1.${DOMAIN}
+export MONGODB_URL              ?= mongodb://mongoroot:secret@mongodb:27017
+export GK_REQUEST_TOKENS        ?= sidetreeToken=ADMIN_TOKEN,vcs_issuer=vcs_issuer_rw_token
```

Comment:
`vcs_issuer=vcs_issuer_rw_token` probably should not be here but rather in `secret.env`, @alialkhalidi what do you think about that?